### ISSUE TITLE
How to set font size of code?

### ISSUE THREAD

**Issue by asifkabani** [deckgo/gatsby-remark-highlight-code#23]
Hi how do I use CSS variables as mentioned, I have never used them before. For example I want to make the font size bigger using the variable  `--deckgo-highlight-code-font-size`.

**Comment by peterpeterparker** [comment]
Hi @asifkabani 

you are right, you can set  the font size using CSS through the variable `--deckgo-highlight-code-font-size`

for example in your website you can add something like

```
deckgo-highlight-code {
    --deckgo-highlight-code-font-size: 4rem;
}
```

Let me know if it works out.

**Comment by peterpeterparker** [comment]
As I did not get any feedback, I am guessing it has been solved. Therefore, I close this question.
Let me know otherwise.

**Comment by simplenotezy** [comment]
> Hi @asifkabani
> 
> you are right, you can set the font size using CSS through the variable `--deckgo-highlight-code-font-size`
> 
> for example in your website you can add something like
> 
> ```
> deckgo-highlight-code {
>     --deckgo-highlight-code-font-size: 4rem;
> }
> ```
> 
> Let me know if it works out.

@peterpeterparker Where do you recommend adding this code? In a CSS module file? The syntax looks very alien to me, and just got started on Gatsby, so explanation in layman terms would be greatly appreciated.
